README.md

```diff
@@ -25,12 +25,29 @@
  1. `cd ~/catkin_ws`
  2. `catkin_make`
 
- ## Running
+ ## Enable Hot Swapping
  1. Connect USB CAN interface (if not already connected)
- 2. Initialize CAN interface
-	 - `~/catkin_ws/src/phoenix-template/scripts/canableStart.sh`
-	 - If there is an error, try running it again.
-	 - If it still doesn't work, check the device name with `ls /dev` and ensure that the device name in the `canableStart.sh` script matches.
- 3. Connect game controller (if not already connected)
- 4. Call launch file
+ 2. Set up permissions
+     - `git clone https://github.com/CrossTheRoadElec/Phoenix-Linux-SocketCAN-Example.git`
+     - `cd ./Phoenix-Linux-SocketCAN-Example/.`
+     - `chmod +x build.sh`
+     - `chmod +x clean.sh`
+     - `chmod +x canableStart.sh`
```

Comment:
Are these steps still necessary if step 3 is followed? If so, let's copy the necessary code into our own configCAN.sh script to be run once.